vignettes/getting_started.Rmd

```diff
@@ -0,0 +1,194 @@
+  Use the package name as repository name.
+  Enter the repository name and optionally a description.
+  Don't add a `README`, `.gitignore` or `LICENSE`.
+
+### Prepare local setup
+
+1. Use `create_package()`.
+  You'll need to provide the package name, title, description and maintainer.
+  Please read the documentation of `create_package()` prior to running the function.
+  The function creates an RStudio project with a the package template in a folder with the name of the package.
+1. Open the newly created RStudio project.
+1. In the _Build pane_, select _More_ > _Configure Build Tools_.
+  Click the _Configure_ button and add a tick to _Install and Restart_.
+  All boxes except _Vignettes_ must be ticked.
+  Close the tow dialogue boxes by clicking the _OK_ buttons (once for each of box).
+1. Open the `DESCRIPTION` file.
+  Add a line `URL: https://inbo.github.io/_your_package_name_` where you replace `_your_package_name_` with the actual repository name.
+1. Open the `README.Rmd` file and update the content.
+  Knit the file and stage the `README.Rmd` and `README.md` files.
+1. Review and commit the changes into an initial commit.
+1. Add at least one function to the package
+1. _Build_ the package.
+1. Run `x <- checklist::check_package()`.
+  Fix any issues that arise during the checks.
+  Repeat the last two steps unless you end up with only a single new note about a _New submission_.
+1. Run `checklist::write_checklist(x)`.
+  This functions can convert some 'new' warnings and notes into 'allowed' warnings and notes.
+1. Run `x <- checklist::check_package()` again to check that only 'allowed' warnings and notes remain.
+  Repeat the last four steps until you get _No problems found_ at the end of the checklist output.
+1. Commit any remaining changes.
+1. Run in the terminal the command `git remote add origin git@github.com:inbo/_your_package_name_.git` where you replace `_your_package_name_` with the actual repository name.
+1. Run in the terminal the command `git push -u origin master`.
+  This will run the checklist test on GitHub for the first time.
+  An orange circle, green thick mark or red cross will appear next to the latest commit or PR.
+
+### Finalise online setup
+
+1. [Create](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token) a personal access token for GitHub.
+Use the scope `public_repo` at minimum.
+If you want to reuse it later, then make sure to store it securely.
+Anyone with your personal access token can access GitHub on your behalf within the scope you gave the token.
+You can revoke old or compromised tokens.
+1. Go to your package repository on GitHub.
+Choose _Settings_ > _Secrets_.
+Add _New repository secret_ with _name_ `PAT` and copy your personal access token in the _value_.
+Save it by clicking the _Add secret_ button.
+1. Run `rorcid::orcid_auth()` and sign in with your [ORCID iD](https://www.orcid.org) if requested.
```

Comment:
Where to run it?  Locally in RStudio in the project folder?  As the previous steps have to be done online in Github, and as the title includes 'online setup', it might be a good idea to specify if it has to be done elsewhere than on Github.